packages/wallets/src/features/cfd/screens/ChangePassword/InvestorPassword/__tests__/MT5ChangeInvestorPasswordInputsScreen.spec.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { PropsWithChildren } from 'react';
+import { useTradingPlatformInvestorPasswordChange } from '@deriv/api-v2';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { WalletButton } from '../../../../../../components/Base';
+import { useModal } from '../../../../../../components/ModalProvider';
+import useDevice from '../../../../../../hooks/useDevice';
```

Comment:
can we use the one from `deriv-com/ui` instead 👀

Reply:
This is out of scope @aizad-deriv, remember these won't go through QA. I don't want surprises 😬